main.css

```diff
@@ -0,0 +1,8 @@
+body {
+  background-color: antiquewhite;
+}
+
+h1 {
+  font-size: 30px;
+  color: brown;
+}
```

Comment:
Nice work on the main.css file!👍🏼
Linter results should be all green. Please fix this linter error on line 8:

`Unexpected missing end-of-source newline   no-missing-end-of-source-newline`

It is being caused because an empty line at the end of the file is missing.